commands/create.go

```diff
@@ -145,7 +146,14 @@ func runCreate(dockerCli command.Cli, in createOptions, args []string) error {
 
 		if in.driver == "kubernetes" {
 			// naming endpoint to make --append works
-			ep = fmt.Sprintf("%s://%s?deployment=%s", in.driver, in.name, in.nodeName)
+			ep = (&url.URL{
+				Scheme: in.driver,
+				Path:   "/" + in.name,
+				RawQuery: (&url.Values{
+					"deployment":  {in.nodeName},
+					"config-file": {os.Getenv("KUBECONFIG")},
```

Comment:
"kubeconfig" rather than "config-file" might be more clear

Reply:
`config-file` from https://docs.docker.com/engine/reference/commandline/context_create/#create-a-context-with-a-docker-and-kubernetes-endpoint 😆 


should follow ? or rename to `kubeconfig` ?

Reply:
updated to `kubeconfig`

Reply:
Hmm, then config-fine makes sense as well... Either is fine to me 😅